src/media/css/footer.styl

```diff
@@ -45,6 +45,11 @@ $support-nav-height = 55px;
             display: block;
         }
     }
+
+    .page-footer-text {
+         color:$action-neutral-hover;
```

Comment:
one less space before `color`, space after the colon `color: $action-neutral-hover;`